mitodl/ocw-hugo-themes
# Add course_description_html field in data.json

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/4377

### Description (What does it do?)
Adds `course_description_html ` field in data.json which uses `.RenderString` function to convert markdown/shortcodes into HTML.


### How can this be tested?
1. Checkout to this branch
2. `yarn start course ibrahims-cat-course ` or any other
3. Go to `http://localhost:3000/data.json` and see the `course_description_html` field populated with HTML version of course description. eg. `"course_description_html": "<p><strong>ibrahims-cat-course</strong></p>\n<p><em><strong>publish</strong></em><strong>Publish</strong></p>\n",`
4. For offline: `yarn build ~/path/ocw-content-rc/ibrahims-cat-course/ ~/path/ocw-hugo-projects/ocw-course-v2/config-offline.yaml`
5. Open `path/ocw-content-rc/ibrahims-cat-course/dist/data.json` and observe HTML is in correct format.
